Gemfile.lock

```diff
@@ -341,6 +341,3 @@ DEPENDENCIES
   turbolinks
   uglifier (>= 1.3.0)
   web-console (~> 2.0)
-
-BUNDLED WITH
-   1.10.4
```

Comment:
Hey @jgreben would you mind updating your bundler gem so that this doesn't get removed in your commits?

Check out [this comment](https://github.com/bundler/bundler/issues/3697#issuecomment-108299463) from one of the bundler devs about why this is happening.